src/display/annotation_layer.js

```diff
@@ -199,6 +199,16 @@ var AnnotationElement = (function AnnotationElementClosure() {
     },
 
     /**
+     * Check does this annotation need HTML element.
```

Comment:
This sentence sounds a bit strange, how about e.g. `Check if the annotation needs a HTML element.` instead?